src/main/core/Resources/modules/account/functional/index.js

```diff
@@ -0,0 +1,10 @@
+import {trans} from '#/main/app/intl/translation'
+
+import {FunctionalMain} from '#/main/core/account/functional/containers/main'
+
+export default {
+  name: 'functional',
+  icon: 'fa fa-fw fa-cogs',
+  label: trans('functional'),
+  component: FunctionalMain
+}
```

Comment:
Don't change it for now, but I think we will make a single `log` app in the account and dashboard.